connector/servicegraphconnector/connector.go

```diff
@@ -13,6 +13,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil"
```

Comment:
This is out of place and causing a lint error:

```
servicegraphconnector/connector.go:28: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
```

Reply:
thanks, This is different from the lint I run locally. Let me check